ubuntudesign/www.ubuntu.com
# Fix cloud events

## Done

* BEMed the classes for news and events to enforce consistency
* updated perl script that builds events to use new classes
* updated js script that builds news to use new classes

## QA

1. go to /cloud
2. update the feed to 'http://www.stmgrts.org.uk/taleo/insights_main.xml' to test
3. see that the fonts are consistent between both news and events

## Issue / Card

Fixes #809 

## Screenshots


* ![large screen](https://cloud.githubusercontent.com/assets/441217/17847112/34015de2-6844-11e6-8abc-75aceb4ce347.png)

* 
![medium screen](https://cloud.githubusercontent.com/assets/441217/17847100/2328a0ca-6844-11e6-95b8-2113388fd57a.png)

* 
![small screen](https://cloud.githubusercontent.com/assets/441217/17847080/07b5ade2-6844-11e6-9207-afc83be83111.png)
